### ISSUE TITLE
File generation in 2.0.0 in a Windows Forms app much slower

### ISSUE THREAD

**Issue by modery** [Rubjerg/Graphviz.NetWrapper#72]
I noticed that version 2.0.0 is significantly slower in file generation (ToDotFile, ToSvgFile, for example) than 1.1.0 when used in a Windows Forms application. Every single ToXYZFile call seems to open a separate console window for a second, I assume to generate the file. This did not happen in 1.1.0

The code example (which generates 20 files) below takes less than a second overall with version 1.1.0. For 2.0.0, it takes a bit more than 20 seconds.
For some of my actual use cases in my application, I noticed that instead of 18 seconds it takes 2 minutes to complete.

Here's how to replicate it:

1. Create a new VSCode project. Use _dotnet new winforms_ to initialize a new simple Windows Forms app
2. update the csproj file to reference the Nuget package (either 1.1.0 or 2.0.0):
```
  <ItemGroup>
    <PackageReference Include="Rubjerg.Graphviz" Version="1.1.0" />
  </ItemGroup>
```
3. Update the Form1.cs file to include the following code right after InitializeComponent();

**1.1.0:**
```
        Console.WriteLine("Starting: " + DateTime.Now);
        for (int i = 0; i < 10; i++)
        {

            RootGraph root = RootGraph.CreateNew("",GraphType.Directed);
            Node nodeA = root.GetOrAddNode("A");
            root.ComputeLayout();
            root.ToDotFile("c:/tmp/out" + i + ".dot");
            RootGraph root2 = RootGraph.FromDotFile("c:/tmp/out" + i + ".dot");
            root2.ComputeLayout();
            root2.ToSvgFile("c:/tmp/dot_out" + i + ".svg");
        }
        Console.WriteLine("Finished: " + DateTime.Now);
```

**2.0.0:**
```
        Console.WriteLine("Starting: " + DateTime.Now);
        for (int i = 0; i < 10; i++)
        {

            RootGraph root = RootGraph.CreateNew(GraphType.Directed, "");
            Node nodeA = root.GetOrAddNode("A");
            root.CreateLayout();
            root.ToDotFile("c:/tmp/out" + i + ".dot");
            RootGraph root2 = RootGraph.FromDotFile("c:/tmp/out" + i + ".dot");
            root2.CreateLayout();
            root2.ToSvgFile("c:/tmp/dot_out" + i + ".svg");
        }
        Console.WriteLine("Finished: " + DateTime.Now);
    }
```

**Comment by chtenb** [comment]
Could this be specific to starting processes in a winforms application?
When I reproduce your code in a unit test it prints:

Starting: 1/19/2024 2:19:40 PM
Finished: 1/19/2024 2:19:43 PM

And after removing the redundant calls to `CreateLayout`:

Starting: 1/19/2024 2:20:19 PM
Finished: 1/19/2024 2:20:20 PM

I don't think we do anything exotic in this library, we just start a process and communicate over stdin/stdout. But since you mention you see a console window opening every call, there would likely be some additional overhead somewhere.

Sounds a bit like https://stackoverflow.com/questions/5377423/hide-console-window-from-process-start-c-sharp which recommends setting `startInfo.UseShellExecute = false;`, but we already do that. Perhaps you could experiment with other options, like `ProcessWindowStyle.Hidden` and see if that resolves your issue?

**Comment by modery** [comment]
I can confirm that running the same code in a console app takes 2 seconds (as expected).

Tested your suggestion, and added the Rubjerg.Graphviz and GraphvizWrapper projects to a test solution. Compiled as is (so same as 2.0.0), and the behaviour is the same (window pops up, generally it's slow).

Once I added the _process.StartInfo.WindowStyle = ProcessWindowStyle.Hidden;_ to the relevant section in GraphvizCommand.Exec, the window didn't pop up again and generation was as fast as before
```
         // Redirect the input/output streams
        process.StartInfo.UseShellExecute = false;
        process.StartInfo.RedirectStandardOutput = true;
        process.StartInfo.RedirectStandardInput = true;
        process.StartInfo.RedirectStandardError = true;
        process.StartInfo.WindowStyle = ProcessWindowStyle.Hidden;
```

Final test was doing the same thing in my actual program, and I can confirm that things worked well once the ProcessWindowStyle.Hidden was set
